src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1469,5 +1469,8 @@ internal enum ErrorCode
         ERR_Merge_conflict_marker_encountered = 8300,
         ERR_InvalidPreprocessingSymbol = 8301,
         ERR_FeatureNotAvailableInVersion7_1 = 8302,
+
+        ERR_RuntimeDoesNotSupportDefaultInterfaceImplementation = 8401,
```

Comment:
Thanks. We have learned already that reserving a noncontiguous block of error ids while developing is good. We can "compress" the error id's later when merging back to master.